requirements/dev.in

```diff
@@ -1,25 +1,25 @@
 -r prod.txt
 
-black==23.11.0
+black==23.12.1
```

Comment:
I have a branch where I moved to Ruff. Should I dust it off and push it up for review?